pandamium_datapack/data/pandamium/functions/triggers/vote_shop.mcfunction

```diff
@@ -1,40 +1,59 @@
 
-execute if score @s vote_shop matches -9 if score @s vote_credits matches 1.. run scoreboard players set @s home_cooldown 0
-execute if score @s vote_shop matches -9 if score @s vote_credits matches 1.. run scoreboard players remove @s vote_credits 1
+execute if score @s vote_shop matches -9 if score @s vote_credits matches 1.. run scoreboard players operation @s temp_2 = @s home_cooldown
+execute if score @s vote_shop matches -9 if score @s vote_credits matches 1.. if score @s temp_2 matches 1.. run scoreboard players set @s home_cooldown 0
+execute if score @s vote_shop matches -9 if score @s vote_credits matches 1.. if score @s temp_2 matches 1.. run scoreboard players remove @s vote_credits 1
 
-execute if score @s vote_shop matches -10 if score @s vote_credits matches 1.. run scoreboard players set @s tpa_cooldown 0
-execute if score @s vote_shop matches -10 if score @s vote_credits matches 1.. run scoreboard players remove @s vote_credits 1
+execute if score @s vote_shop matches -10 if score @s vote_credits matches 1.. run scoreboard players operation @s temp_2 = @s tpa_cooldown
+execute if score @s vote_shop matches -10 if score @s vote_credits matches 1.. if score @s temp_2 matches 1.. run scoreboard players set @s tpa_cooldown 0
+execute if score @s vote_shop matches -10 if score @s vote_credits matches 1.. if score @s temp_2 matches 1.. run scoreboard players remove @s vote_credits 1
 
-execute if score @s vote_shop matches ..-1 unless score @s vote_credits = @s temp_1 run tellraw @s [{"text":"Purchase sucessful! ","color":"green"},{"text":"You currently have ","color":"green"},{"score":{"name":"@s","objective":"vote_credits"},"color":"aqua"},{"text":" credits left.\n","color":"green"}]
-execute if score @s vote_shop matches ..-1 if score @s vote_credits = @s temp_1 run tellraw @s [{"text":"Error: You do not have enough credits!","color":"red"}]
+execute if score @s vote_shop matches -10..-9 run scoreboard players set @s temp_3 0
+
+execute if score @s temp_3 matches ..35 if score @s vote_shop matches ..-1 unless score @s vote_credits = @s temp_1 run tellraw @s ["",{"text":"Purchase successful! ","color":"aqua","bold":true},"\n",[{"text":"You have ","color":"green","italic":true},{"score":{"name":"@s","objective":"vote_credits"},"color":"aqua","italic":false}," vote credits left."]]
+execute if score @s temp_3 matches ..35 if score @s vote_shop matches ..-1 if score @s vote_credits = @s temp_1 unless score @s temp_2 matches 0 run tellraw @s {"text":"Error: You do not have enough credits!","color":"red"}
+execute if score @s temp_3 matches ..35 if score @s vote_shop matches -10..-9 if score @s temp_2 matches 0 run tellraw @s {"text":"Error: You do not have a cooldown!","color":"red"}
+execute if score @s temp_3 matches 36.. if score @s vote_shop matches ..-1 unless score @s vote_shop matches -10..-9 run tellraw @s {"text":"Error: Your inventory is full!","color":"red"}
```

Comment:
Instead of `if ... matches 36..` I'd prever `unless ... matches ..35` to make it more explicit it's the opposite of the cases above